js/ui/save-html.js

```diff
@@ -75,6 +80,17 @@ define(
                 }
                 ui.freshModal("Save Snapshot", $div);
             }
+        // Create and download an EPUB 3 version of the content
+        // Using (by default) the EPUB 3 conversion service set up at labs.w3.org/epub-generator
+        // For more details on that service, see https://github.com/iherman/respec2epub
+        ,   makeEPubHref: function(){
+                if (!conf.epubGenTool) {
+                    conf.epubGenTool = "https://labs.w3.org/epub-generator/cgi-bin/epub-generator.py";
```

Comment:
Is it worth making this an option? Are there other online services that do EPUB generation from HTML?